src/EnergyPlus/SingleDuct.cc

```diff
@@ -2358,9 +2358,9 @@ namespace SingleDuct {
 						if ( PltSizHeatNum > 0 ) {
 							CoilInTemp = TermUnitFinalZoneSizing( CurZoneEqNum ).DesHeatCoilInTempTU;
 							DesMassFlow = StdRhoAir * TermUnitSizing( CurZoneEqNum ).AirVolFlow;
-							DesZoneHeatLoad = CalcFinalZoneSizing( CurZoneEqNum ).DesHeatLoad * CalcFinalZoneSizing( CurZoneEqNum ).HeatSizingFactor;
```

Comment:
Changed to use ACTUAL final zone sizing data. I think all equipment models should use the final sizing array and not mix and match arrays. Probably OK, but starting a trend to move to the same sizing array for all models.